diff --git a/proxmox-frr/src/ser/serializer.rs b/proxmox-frr/src/ser/serializer.rs
--- a/proxmox-frr/src/ser/serializer.rs
+++ b/proxmox-frr/src/ser/serializer.rs
@@ -8,51 +8,51 @@ use proxmox_sortable_macro::sortable;
 pub static TEMPLATES: [(&str, &str); 12] = sorted!([
     (
         "fabricd.jinja",
-        include_str!("/usr/share/proxmox-frr/templates/fabricd.jinja"),
+        include_str!("/src/build/proxmox-ve-rs/proxmox-frr-templates/templates/fabricd.jinja"),
     ),
     (
         "bgpd.jinja",
-        include_str!("/usr/share/proxmox-frr/templates/bgpd.jinja")
+        include_str!("/src/build/proxmox-ve-rs/proxmox-frr-templates/templates/bgpd.jinja")
     ),
     (
         "isisd.jinja",
-        include_str!("/usr/share/proxmox-frr/templates/isisd.jinja"),
+        include_str!("/src/build/proxmox-ve-rs/proxmox-frr-templates/templates/isisd.jinja"),
     ),
     (
         "ospfd.jinja",
-        include_str!("/usr/share/proxmox-frr/templates/ospfd.jinja"),
+        include_str!("/src/build/proxmox-ve-rs/proxmox-frr-templates/templates/ospfd.jinja"),
     ),
     (
         "bgp_router.jinja",
-        include_str!("/usr/share/proxmox-frr/templates/bgp_router.jinja"),
+        include_str!("/src/build/proxmox-ve-rs/proxmox-frr-templates/templates/bgp_router.jinja"),
     ),
     (
         "interface.jinja",
-        include_str!("/usr/share/proxmox-frr/templates/interface.jinja"),
+        include_str!("/src/build/proxmox-ve-rs/proxmox-frr-templates/templates/interface.jinja"),
     ),
     (
         "access_lists.jinja",
-        include_str!("/usr/share/proxmox-frr/templates/access_lists.jinja"),
+        include_str!("/src/build/proxmox-ve-rs/proxmox-frr-templates/templates/access_lists.jinja"),
     ),
     (
         "prefix_lists.jinja",
-        include_str!("/usr/share/proxmox-frr/templates/prefix_lists.jinja"),
+        include_str!("/src/build/proxmox-ve-rs/proxmox-frr-templates/templates/prefix_lists.jinja"),
     ),
     (
         "route_maps.jinja",
-        include_str!("/usr/share/proxmox-frr/templates/route_maps.jinja"),
+        include_str!("/src/build/proxmox-ve-rs/proxmox-frr-templates/templates/route_maps.jinja"),
     ),
     (
         "ip_routes.jinja",
-        include_str!("/usr/share/proxmox-frr/templates/ip_routes.jinja"),
+        include_str!("/src/build/proxmox-ve-rs/proxmox-frr-templates/templates/ip_routes.jinja"),
     ),
     (
         "protocol_routemaps.jinja",
-        include_str!("/usr/share/proxmox-frr/templates/protocol_routemaps.jinja"),
+        include_str!("/src/build/proxmox-ve-rs/proxmox-frr-templates/templates/protocol_routemaps.jinja"),
     ),
     (
         "frr.conf.jinja",
-        include_str!("/usr/share/proxmox-frr/templates/frr.conf.jinja"),
+        include_str!("/src/build/proxmox-ve-rs/proxmox-frr-templates/templates/frr.conf.jinja"),
     ),
 ]);
 
