diff --git a/pve-rs/debian/control b/pve-rs/debian/control
--- a/pve-rs/debian/control
+++ b/pve-rs/debian/control
@@ -4,7 +4,6 @@ Priority: optional
 Build-Depends: debhelper-compat (= 13),
                dh-cargo,
                cargo:native <!nocheck>,
-               libproxmox-rs-perl (>= 0.3.5),
                librust-anyhow-1+default-dev,
                librust-base32-0.4+default-dev,
                librust-hex-0.4+default-dev,
