diff --git a/src/bin/proxmox-daily-update.rs b/src/bin/proxmox-daily-update.rs
--- a/src/bin/proxmox-daily-update.rs
+++ b/src/bin/proxmox-daily-update.rs
@@ -1,5 +1,6 @@
 use anyhow::Error;
 use serde_json::json;
+use std::io::Write;
 
 use proxmox_notify::context::pbs::PBS_CONTEXT;
 use proxmox_router::{cli::*, ApiHandler, RpcEnvironment};
@@ -111,15 +112,42 @@ async fn run(rpcenv: &mut dyn RpcEnvironment) -> Result<(), Error> {
     do_update(rpcenv).await
 }
 
+struct ContainerLogger;
+
+static LOGGER: ContainerLogger = ContainerLogger;
+
+impl log::Log for ContainerLogger {
+    fn enabled(&self, metadata: &log::Metadata<'_>) -> bool {
+        metadata.level() <= log::Level::Info
+    }
+
+    fn log(&self, record: &log::Record<'_>) {
+        if !self.enabled(record.metadata()) {
+            return;
+        }
+
+        if let Ok(mut output) = std::fs::OpenOptions::new()
+            .write(true)
+            .open("/proc/1/fd/1")
+        {
+            let _ = writeln!(output, "{}: {}", record.level(), record.args());
+        }
+    }
+
+    fn flush(&self) {}
+}
+
+fn init_container_logger() -> Result<(), log::SetLoggerError> {
+    log::set_logger(&LOGGER)?;
+    log::set_max_level(log::LevelFilter::Info);
+
+    Ok(())
+}
+
 fn main() {
     proxmox_backup::tools::setup_safe_path_env();
 
-    // We need to use the tasklog layer here because we call a workertask.
-    proxmox_log::Logger::from_env("PBS_LOG", proxmox_log::LevelFilter::INFO)
-        .journald_on_no_workertask()
-        .tasklog_pbs()
-        .init()
-        .expect("unable to initialize logger");
+    init_container_logger().expect("unable to initialize logger");
 
     let mut rpcenv = CliEnvironment::new();
     rpcenv.set_auth_id(Some(String::from("root@pam")));
